### PR TITLE
Shiqi

### PR DESCRIPTION
**Pull request by shiqio3o**
merge to main

### CODE REVIEW

File: src/data_access/FileFolderDataAccessObject.java [hunk 1]
```diff
@@ -102,4 +100,9 @@ public boolean existsByName(String identifier) {
     public Folder get(String folderName) {
         return folders.get(folderName);
     }
+
+//    @Override
```
Comment (yuxizzz) [inline src/data_access/FileFolderDataAccessObject.java]: Do you still need this method in folder DAO?

File: src/use_case/delete_folder/DeleteFolderUserDataAccessInterface.java [hunk 2]
```diff
@@ -4,4 +4,6 @@
 
 public interface DeleteFolderUserDataAccessInterface {
     String deleteFolder(String folderName);
+
+    boolean existsByName(String identifier);
```
Comment (yuxizzz) [inline src/use_case/delete_folder/DeleteFolderUserDataAccessInterface.java:8]: why do you add existsByName in DeteleFolder? when you press the button on the folder, the folder should exist. there is no need to add this method for checking

Comment (shiqio3o) [reply src/use_case/delete_folder/DeleteFolderUserDataAccessInterface.java:8]: this can delete, i will update this

Comment (yuxizzz) [inline src/use_case/delete_folder/DeleteFolderUserDataAccessInterface.java:8]: great!

**Review by yuxizzz — CHANGES_REQUESTED** [review]
change method in src/use_case/delete_folder/DeleteFolderUserDataAccessInterface.java

**Review by yuxizzz — APPROVED** [review]

**Pull request merged**